### PR TITLE
Many backward compatible updates and additions

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: resources.nix [hunk 1]
```diff
@@ -367,6 +383,39 @@
         doCheck = false;
       };
 
+      hermes_1_5_1 = pkgs.rustPlatform.buildRustPackage {
```
Comment (romac) [inline resources.nix:386]: Let's instead update the `hermes` flake instead of adding a new one

File: resources/utilities.nix [hunk 2]
```diff
@@ -76,4 +76,10 @@ in {
       install_name_tool -add_rpath "${libwasmvm}/lib" $out/bin/${binName}
     ''
     else null;
+  
+  darwin-deps = pkgs.lib.optional pkgs.stdenv.isDarwin (with pkgs;
+    with darwin.apple_sdk.frameworks; [
+      Security
+      SystemConfiguration
```
Comment (romac) [inline resources/utilities.nix:83]: Is this framework really necessary? `otool -L` does not show it for me:

```
/System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 60420.101.2)
/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1971.0.0)
/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1319.100.3)
```

**Review by romac — DISMISSED** [review]
Awesome, thanks for this! Just one minor change needed and a question inline.

File: resources/utilities.nix (hunk 2 above)
Comment (dzmitry-lahoda) [reply resources/utilities.nix:83]: Security 100% - fails to compile on m1

**Comment by romac** [comment]
Closed in favor of #141

**Pull request closed without merging**